operator/utils/utils.go

```diff
@@ -84,17 +86,27 @@ func AppendTLSOptionsArgs(opts *k8upv1.TLSOptions, prefixArgName ...string) []st
 }
 
 func AttachTLSVolumes(volumes *[]k8upv1.RunnableVolumeSpec) []corev1.Volume {
```

Comment:
At this point we should probably rename the function.

```suggestion
func AttachEmptyDirVolumes(volumes *[]k8upv1.RunnableVolumeSpec) []corev1.Volume {
```